packages/fuels-core/src/types/transaction_builders.rs

```diff
@@ -205,7 +183,7 @@ impl_tx_trait!(ScriptTransactionBuilder, ScriptTransaction);
 impl_tx_trait!(CreateTransactionBuilder, CreateTransaction);
 
 impl ScriptTransactionBuilder {
-    fn convert_to_fuel_tx(self, base_offset: usize) -> ScriptTransaction {
+    fn convert_to_fuel_type(self, base_offset: usize) -> Script {
```

Comment:
I don't mind the renaming, but maybe the Builders should follow the naming convention then

Reply:
I needed a way to construct the underlying fuel's type (in this case `Script`) and then later I make the `ScriptTransaction`